recipe/meta.yaml

```
@@ -24,9 +24,17 @@ source:
   sha256: a475bc85cfc204aff825e3e5d97d7c1993d3f2cfec22a87e76b8d596fc496a02  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx]
 
+test:
+  commands:
+    - test -L $PREFIX/lib/libcusparse.so.{{ version }}                                  # [linux]
+    - test -L $PREFIX/lib/libcusparse.so.{{ version.split(".")[0] }}                    # [linux]
+    - test -L $PREFIX/{{ target_name }}/lib/libcusparse.so.{{ version }}                # [linux]
+    - test -f $PREFIX/{{ target_name }}/lib/libcusparse.so.{{ version.split(".")[0] }}  # [linux]
```

Updating flags here, and match order of other feedstocks.
```suggestion
    - test -L $PREFIX/{{ target_name }}/lib/libcusparse.so.{{ version.split(".")[0] }}  # [linux]
    - test -f $PREFIX/{{ target_name }}/lib/libcusparse.so.{{ version }}                # [linux]
```

---

Good catch. But I think even if we fix that, we're still missing the "targets" part of `$PREFIX/targets/{{ target_name }}`. I'll adapt this suggestion and apply it.
```suggestion
    - test -L $PREFIX/targets/{{ target_name }}/lib/libcusparse.so.{{ version.split(".")[0] }}  # [linux]    
    - test -f $PREFIX/targets/{{ target_name }}/lib/libcusparse.so.{{ version }}                # [linux]
```